### PR TITLE
[docs] Say how to switch to apt install on linux

### PR DESCRIPTION
**Pull request by rfay**
## The Problem/Issue/Bug:

Docs: Explain how to switch to the apt install from install_ddev.sh or homebrew.

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/4025"></a>

### CODE REVIEW

File: docs/content/users/install/ddev-installation.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Docker or an alternative is required before anything will work with DDEV. This i
   
     In the future you can update as usual, with `sudo apt update && sudo apt upgrade`.
 
+    If you previously used `install_ddev.sh` to install DDEV, you can just `sudo rm /usr/local/bin/ddev` to remove the previous version. If you previously used homebrew to install DDEV, you can just `brew unlink ddev` to get rid of the homebrew version.
```
Comment (gilbertsoft) [inline docs/content/users/install/ddev-installation.md:49]: There are 3 artifacts which are copied by the script see https://github.com/drud/ddev/blob/master/scripts/install_ddev.sh#L12, I've removed them all while switching.

Comment (rfay) [reply docs/content/users/install/ddev-installation.md:49]: You can just click the "Add a suggestion" button to improve the text :)

Comment (gilbertsoft) [inline docs/content/users/install/ddev-installation.md:49]:
```suggestion
    If you previously used `install_ddev.sh` to install DDEV, you can just `sudo rm /usr/local/bin/ddev /usr/local/bin/mkcert /usr/local/bin/macos_ddev_nfs_setup.sh` to remove the previous version. If you previously used homebrew to install DDEV, you can just `brew unlink ddev` to get rid of the homebrew version.
```

Comment (gilbertsoft) [reply docs/content/users/install/ddev-installation.md:49]: Yes I know. Just wasn't sure if this was intended to omit these files ;)

**Pull request merged**